src/zenml/zen_stores/migrations/versions/b0e68bb87611_add_api_key_table.py

```diff
@@ -0,0 +1,64 @@
+"""add api key table [b0e68bb87611].
+
+Revision ID: b0e68bb87611
+Revises: 47f55c2589f6
+Create Date: 2023-09-20 16:54:02.154481
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "b0e68bb87611"
+down_revision = "47f55c2589f6"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
```

Comment:
I'm so used to see batch_op operations in all of our migration scripts. I assume you tested this one and it works fine on sqlite?

Reply:
`batch_op` is only used to update existing tables, not to create new ones.